unit_testing/test_deepseg.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# pytest unit tests for spinalcordtoolbox.deepseg
+
+
+import os
+
+import spinalcordtoolbox as sct
+import spinalcordtoolbox.deepseg.core
+import spinalcordtoolbox.deepseg.models
+
+
+def test_model_dict():
+    """
+    Make sure all fields are present in each model.
+    :return:
+    """
+    for key, value in sct.deepseg.models.MODELS.items():
+        assert('url' in value)
+        assert('description' in value)
+        assert('default' in value)
```

Comment:
Not a suggestion, just a comment: this is reaching for static typing. You're wanting something like:

```
class Model:
  url: string = None
  description: string = None
  default: bool = False
```

I think the pythonic way to handle this is, well, either to impose static typing with `mypy`, or to not worry about it at all and let the tests catch these issues.

Reply:
cool! good to know 👍

Reply:
Why typecheck this but not typecheck the metadata `.json`s? It's something to think about.

Reply:
I've added a test that downloads all models 85bdc4534121d1690f0087dc547eabec77692814, but i would wait for checking json fields because this is still an [open discussion](https://github.com/neuropoly/ivado-medical-imaging/issues/193)